Makefile

```diff
@@ -0,0 +1,10 @@
+build:
+	python setup.py sdist bdist_wheel --universal
+
+clean-build:
+	rm --force --recursive build/
+	rm --force --recursive dist/
+	rm --force --recursive *.egg-info
+
+upload:
+  twine upload dist/*
```

Comment:
add a dependency chain here. 


```Makefile
.PHONY clean-build build upload

clean-build: 

build: clean-build

upload: build
```

this way we can just say upload and are always sure that the build is clean.